src/aig-common/modules/standard/components/ipp-lot-category/lot-category-form/lot-category-form.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { FuseProgressBarService } from '@fuse/components/progress-bar/progress-bar.service';
+import { ItalianPublicProcurementLotCategoryDTO, ItalianPublicProcurementLotCategoryResourceService } from 'aig-standard';
+import { EventService } from 'aig-common/event-manager/event.service';
+
+@Component({
+    selector: 'aig-lot-category-form',
+    templateUrl: './lot-category-form.component.html',
+    styleUrls: ['./lot-category-form.component.scss']
+})
+export class AigLotCategoryNewUpdateFormComponent implements OnInit {
+
+    private step: any = {
+        form: true,
+        loading: false,
+        complete: false
+    };
+
+    constructor(
+        private _formBuilder: FormBuilder,
+        private _fuseProgressBarService: FuseProgressBarService,
+        private _snackBar: MatSnackBar,
+        private categoryResourceService: ItalianPublicProcurementLotCategoryResourceService,
```

Comment:
change to:
`private ippLotCategoryResourceService: ItalianPublicProcurementLotCategoryResourceService,`